spark/src/main/java/com/nflabs/zeppelin/spark/SparkSqlInterpreter.java

```diff
@@ -76,13 +76,20 @@ public void open() {
     this.maxResult = conf.getInt("ZEPPELIN_SPARK_MAX_RESULT",
         "zeppelin.spark.maxResult",
         Integer.parseInt(getProperty("zeppelin.spark.maxResult")));
+    openSparkInterpreter(true);
```

Comment:
Also, we need to make sure it is OK to reopen spark interpreter while it's already opened.